ynr/apps/candidates/helpers/helpers.py

```diff
@@ -0,0 +1,20 @@
+from uk_election_timetables.calendars import Country
+from uk_election_timetables.election_ids import from_election_id
+
+
+def get_election_timetable(slug, territory):
+    country = {
```

Comment:
I wonder if at some point it would be worth discussing standardising these short codes? Sometimes we use "SCO" and "WAL", but sometimes we use "SCT" and "WLS". Nothing for you to fix right now, just some food for thought 🙂